src/data/screenshots_de.json

```diff
@@ -6,13 +6,13 @@
     },
     "section-main": {
         "headline": {
-            "title": "Machen Sie sich selbst ein Bild (Version 2.24)"
+            "title": "Machen Sie sich selbst ein Bild (Version 2.25)"
         },
         "content": {
             "textblock": [
-                "Verschaffen Sie sich mithilfe von Screenshots einen Überblick über die Funktionen, die die Corona-Warn-App mit Version 2.24 (verfügbar seit 29. Juni 2022) bietet. Bitte beachten Sie, dass die Screenshots für Android und iOS designbedingt leicht voneinander abweichen können.",
+                "Verschaffen Sie sich mithilfe von Screenshots einen Überblick über die Funktionen, die die Corona-Warn-App mit Version 2.25 (verfügbar seit ??. ??? ????) bietet. Bitte beachten Sie, dass die Screenshots für Android und iOS designbedingt leicht voneinander abweichen können.",
                 "Mit diesem Update stellen wir neue und erweiterte Funktionen zur Verfügung:",
-                "<b>Zertifikate exportieren:</b> Sie können in einem Schritt alle in Ihrer App vorhandenen Zertifikate in einem gemeinsamen PDF-Dokument speichern.",
+                "<b>Verhaltensempfehlungen aktualisiert:</b> Die Verhaltensempfehlungen bei erhöhten Risiko wurden aktualisiert. Das betrifft unter anderem Empfehlungen zur Reduktion von Kontakten oder dem Tragen von Masken. Angepasst wurden auch die Empfehlungen bei niedrigem Risiko.",
```

Comment:
"bei erhöhten Risiko" => "bei erhöhtem Risiko"

Compare to Android app text:

```
    <string-array name="new_release_body">
        <item>Die Verhaltensempfehlungen bei erhöhtem Risiko wurden aktualisiert. Das betrifft unter anderem Empfehlungen zur Reduktion von Kontakten oder dem Tragen von Masken. Angepasst wurden auch die Empfehlungen bei niedrigem Risiko.</item>
    </string-array>
```

Reply:
Fixed in https://github.com/corona-warn-app/cwa-website/pull/3024/commits/6d345b82c927b90963b16bf8a226ac525016a904, thanks!